blob-polyfill.ts

```diff
@@ -0,0 +1,99 @@
+import { getArrayBufferForBlob } from "react-native-blob-jsi-helper";
+import { ReadableStream } from "web-streams-polyfill";
```

Comment:
We already have https://github.com/browserify/stream-browserify and readable-stream. do we need to add this or can we make do with the stream polyfills we already have?

Reply:
I initially had tried that, but it's incompatible with this part of Web5 https://github.com/TBD54566975/web5-js/blob/v0.7.11/packages/web5-user-agent/src/utils.ts#L8